installing/installing_platform_agnostic/installing-platform-agnostic.adoc

```diff
@@ -0,0 +1,136 @@
+[id="installing-platform-agnostic"]
+= Installing a cluster on any platform
+include::modules/common-attributes.adoc[]
+:context: installing-platform-agnostic
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+any infrastructure that you provision, including virtualization and cloud environments.
+
+[IMPORTANT]
+====
+ Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]
+before you attempt to install an {product-title} cluster in
+virtualized or cloud environments.
+====
+
+== Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-infrastructure-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-network-user-infra.adoc[leveloffset=+2]
+
+include::modules/installation-dns-user-infra.adoc[leveloffset=+2]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+
+include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+include::modules/installation-three-node-cluster.adoc[leveloffset=+1]
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
+
+[id="creating-machines-platform-agnostic"]
+== Creating {op-system-first} machines
```

Comment:
Per my understanding, OSDOCS-1697 is talking about baremetal install on all kinds of cloud platforms. If that, I think The approach of creating rhcos machines is the most different things compared with the installation on the real baremetal.
On the real baremetal machines, user have to use PXE to install rhcos on these real baremtal machines to continue the following ocp install. While on other cloud platforms, user can directly upload the existing images to cloud platforms, then use these image to boot up machines to continue ocp install. E.g:
https://deploy-preview-28881--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-rhcos-ami_installing-aws-user-infra
https://deploy-preview-28881--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user.html#installation-osp-creating-image_installing-openstack-user
Even on some platform, user can not run pxe install there, because platform does not support that.

Reply:
@jianlinliu, is this a reasonable update?

Reply:
LGTM.